docs/archive.md

```diff
@@ -0,0 +1,26 @@
+# Data Archiving
+
+When the crawls are complete, the data is archived to several places:
+
+- Internet Archive (sql)
+- HTTP Archive (sql, csv)
+- Google Storage (har)
+- BigQuery (har)
+
```

Comment:
FWIW, I don't think we need to document every surface here, as much as.. points folks towards the canonical place they should prefer and can rely on. For that, I propose we use gs://.